pom.xml

```diff
@@ -26,9 +26,36 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.uber.tchannel</groupId>
-    <artifactId>tchannel</artifactId>
+    <artifactId>tchannel-java</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
     <packaging>pom</packaging>
-    <version>1.0-SNAPSHOT</version>
+
+    <name>tchannel-java</name>
+    <description>Network multiplexing and framing protocol for RPC</description>
+    <url>https://github.com/uber/tchannel-java</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://www.opensource.org/licenses/mit-license.php</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Will Salisbury</name>
```

Comment:
can we have this as tchannel contributors (its hard to manage all developer contributions like this going forward) and instead of your personal email some general google group where people can actually ask questions ?